oauth2server/apps/credentials/fixtures/ons_credentials.json

```diff
@@ -0,0 +1,61 @@
+[
```

Comment:
1. do we understand the django salt-hash algorithm that's used to generate these? I'm guessing they aren't salted correctly as they all have the same hash :o)
2. do we have a record of what these un-hashed passwords are. I'm assuming they are very much development passwords.
3. we must ensure we provision environment specific versions of this file or overridden the values per environment deployment.